packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -488,7 +496,27 @@
   }
 
   .is-calendar-focused {
-    outline: var(--terra-date-picker-month-focus-outline, 2px dashed #000);
-    outline-offset: var(--terra-date-picker-month-focus-outline-offset, 0);
+    &--keyboard-focus {
+      outline: var(--terra-date-picker-day-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-date-picker-day-month-focus-outline-offset, 0);
+    }
+  }
+
+  .react-datepicker-day {
+    &--selected-border {
+      outline: var(--terra-date-picker-day-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-date-picker-day-month-focus-outline-offset, 0);
```

Comment:
Should `month` be removed from `--terra-date-picker-day-month-focus-outline-offset` to be consistent with `--terra-date-picker-day-focus-outline`? Same on line 501.

Reply:
Fixed 6797a7f